File: src/gui/maisonH.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package gui;

/**
 *
 * @author CC
 */

public class maisonH {

    static int size() {
        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
    }
    private String nom,localisation,description,image_maison;
    private int id_maison;
    private float prix;
    

      public maisonH(String nom, String localisation, String description, float prix, String image_maison) {
        this.nom = nom;
        this.localisation = localisation;
        this.description = description;
        this.prix = prix;
        this.image_maison = image_maison;
    }
    
    
    public maisonH(int id_maison,String nom, String localisation, String description, float prix, String image_maison) {
        this.id_maison=id_maison ;
        this.nom = nom;
        this.localisation = localisation;
        this.description = description;
        this.prix = prix;
        this.image_maison = image_maison;
    }

    
     public int getId_maison() {
        return id_maison;
    }
    public String getNom() {
        return nom;
    }

    

    public String getLocalisation() {
        return localisation;
    }

    public String getDescription() {
        return description;
    }

    public float getPrix() {
        return prix;
    }
    
    public String getImage_maison() {
        return image_maison;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

   
    public void setLocalisation(String localisation) {
        this.localisation = localisation;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setPrix(float prix) {
        this.prix = prix;
    }
        public void setId_maison(int id_maison) {
        this.id_maison = id_maison;
    }
        
     public void setImage_maison(String image_maison) {
        this.image_maison = image_maison;
    }

    @Override
    public String toString() {
        return "maisonH{" + "nom=" + nom + ", localisation=" + localisation + ", description=" + description + ", image_maison=" + image_maison + ", id_maison=" + id_maison + ", prix=" + prix + '}';
    }


    
}

File: Desktop/gestionevenement/src/Entity/Maisonh.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Maisonh
 *
 * @ORM\Table(name="maisonh")
 * @ORM\Entity
 */
class Maisonh
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_maison", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idMaison;

    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=50, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="localisation", type="string", length=1000, nullable=false)
     */
    private $localisation;

    /**
     * @var string
     *
     * @ORM\Column(name="description", type="string", length=1000, nullable=false)
     */
    private $description;

    /**
     * @var float
     *
     * @ORM\Column(name="prix", type="float", precision=10, scale=0, nullable=false)
     */
    private $prix;

    /**
     * @var string
     *
     * @ORM\Column(name="image_maison", type="string", length=100, nullable=false)
     */
    private $imageMaison;

    public function getIdMaison(): ?int
    {
        return $this->idMaison;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getLocalisation(): ?string
    {
        return $this->localisation;
    }

    public function setLocalisation(string $localisation): self
    {
        $this->localisation = $localisation;

        return $this;
    }

    public function getDescription(): ?string
    {
        return $this->description;
    }

    public function setDescription(string $description): self
    {
        $this->description = $description;

        return $this;
    }

    public function getPrix(): ?float
    {
        return $this->prix;
    }

    public function setPrix(float $prix): self
    {
        $this->prix = $prix;

        return $this;
    }

    public function getImageMaison(): ?string
    {
        return $this->imageMaison;
    }

    public function setImageMaison(string $imageMaison): self
    {
        $this->imageMaison = $imageMaison;

        return $this;
    }


}
